config/manager/webhook.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: apps/v1
+kind: DaemonSet
```

Comment:
Would this be more appropriate as a Deployment with replicas=2?  DaemonSet seems like overkill.

Reply:
Yeah I could change it. I guess we don't know how many worker nodes all the users of DWS will have.

Reply:
Does it have to be on every worker node, or does it just need a single replica?

Reply:
Well, a deployment with replicas=2 isn't enough to guarantee that the two pods are on different nodes.  You'd also have to add pod anti-affinity...

but I still think daemonset is overkill.